playwright/src/main/java/com/microsoft/playwright/junit/impl/APIRequestContextExtension.java

```diff
@@ -7,9 +7,14 @@
 
 import static com.microsoft.playwright.junit.impl.ExtensionUtils.isParameterSupported;
 
-public class APIRequestContextExtension implements ParameterResolver, AfterAllCallback {
+public class APIRequestContextExtension implements ParameterResolver, BeforeEachCallback, AfterAllCallback {
```

Comment:
I believe it should be `BeforeTestExecutionCallback` to avoid the problem described below.

Reply:
Yes, you are right.